core-common/pom.xml

```diff
@@ -12,14 +12,15 @@
   <description>Common resources for the Arrowhead core services</description>
 
   <properties>
-    <client.skeleton>core-client-skeleton</client.skeleton>
+    <core.common.essentials>core-common-essentials-java-spring</core.common.essentials>
```

Comment:
should be: `arrowhead-core-common-essentials-java-spring`

See:
![JIRA](https://user-images.githubusercontent.com/48867251/125623566-dcae2089-da98-40cc-99e3-8983e0ba08c9.JPG)

Reply:
ok, I fixed the name.